integrations/clickup/clickup/utils.py

```diff
@@ -0,0 +1,7 @@
+import enum
+
+
+class CustomProperties(enum.StrEnum):
+    """Creating custom properties for the Clickup integration."""
+
+    TEAM_ID = "__team_id"
```

Comment:
This could be simplified by making it an enum

Reply:
Done